resolver-dns/src/main/java/io/netty/resolver/dns/DnsServerAddressStreamProviders.java

```diff
@@ -66,9 +66,11 @@ public Object run() {
                 } else {
                     throw (Throwable) maybeProvider;
                 }
+            } catch (ClassNotFoundException cause) {
+                LOGGER.warn("Can not find {} in the classpath, fallback to system defaults. {}",
+                        MACOS_PROVIDER_CLASS_NAME, "This may result in incorrect DNS resolutions on MacOS.");
```

Comment:
nit:  why you give two arguments here ? Just use:

```
"Can not find {} in the classpath, fallback to system defaults. This may result in incorrect DNS resolutions on MacOS.", MACOS_PROVIDER_CLASS_NAME 
```

Reply:
It doesn't fit into 120 line length and requires string concatenation. I was trying to avoid that with 2nd logger argument. But actually, I think javac/jvm should be able to see that there are 2 string constants and optimize their concatenation in advance.